polling_stations/apps/feedback/tests/test_feedback.py

```diff
@@ -0,0 +1,77 @@
+# from django.test import TestCase
+# from django.core.management import call_command
+
+# # from feedback.models import Feedback
+# from polling_stations.apps.councils.tests.factories import CouncilFactory
+
+# """
+# Tests for the feedback app
+
+# """
+
+
+# class FeedbackTestCase(TestCase):
+#     @classmethod
+#     def setUpTestData(cls):
+#         CouncilFactory(
+#             council_id="X01",
+#             identifiers=["X01"],
+#             geography__geography=None,
+#         )
+
+#         call_command(  # Hack to avoid converting all fixtures to factories
+#             "loaddata",
+#             "polling_stations/apps/data_finder/fixtures/test_single_address_single_polling_station.json",
+#             verbosity=0,
+#         )
+
+#     def test_feedback_form(self):
```

Comment:
I think if you add `@override_settings(EVERY_ELECTION={"CHECK": False, "HAS_ELECTION": True})` as a decorator on the method it should work

Reply:
That worked, thanks!